### DISCUSSION TITLE
v0.3.0

### DISCUSSION THREAD

**Discussion by mitchelbaker-cisa** [cisagov/ScubaGoggles#476] [Announcements]
## Major Changes 
* Make it so that the scubagoggles reports link to the baselines by @jfevang in https://github.com/cisagov/ScubaGoggles/pull/323
* Add Detailed report messages for Drive&Docs by @snarve in https://github.com/cisagov/ScubaGoggles/pull/300
* Orchestrator: convert to class implementation (issue 167) by @rlxdev in https://github.com/cisagov/ScubaGoggles/pull/333
* Implement Gmail 19.2 and 19.3 in rego by @adhilto in https://github.com/cisagov/ScubaGoggles/pull/338
* Implement group checks for Meet (issue 320) by @rlxdev in https://github.com/cisagov/ScubaGoggles/pull/329
* New detailed report message for Groups by @snarve in https://github.com/cisagov/ScubaGoggles/pull/301
* Reporter: convert to class implementation (issue 166) by @rlxdev in https://github.com/cisagov/ScubaGoggles/pull/342
* Default Safe variable is incorrect in many cases by @snarve in https://github.com/cisagov/ScubaGoggles/pull/343
* Add Omit to the Summary Counts by @adhilto in https://github.com/cisagov/ScubaGoggles/pull/346
* Add functional smoke tests and automated testing workflow by @mitchelbaker-cisa in https://github.com/cisagov/ScubaGoggles/pull/336
* Rename the baseline markdown files by @adhilto in https://github.com/cisagov/ScubaGoggles/pull/354
* update supported Python version  by @aormu in https://github.com/cisagov/ScubaGoggles/pull/356
* Add config file support for existing parameters by @adhilto in https://github.com/cisagov/ScubaGoggles/pull/413
* Add support for user input and documentation improvements to smoke testing workflow by @mitchelbaker-cisa in https://github.com/cisagov/ScubaGoggles/pull/416
* Update Calendar policy 4.1 to detect inherited settings by @snarve in https://github.com/cisagov/ScubaGoggles/pull/471
* Update sample report for v0.3.0 by @adhilto in https://github.com/cisagov/ScubaGoggles/pull/472

## Documentation
* Update DownloadAndInstall.md by @LaurenBassett in https://github.com/cisagov/ScubaGoggles/pull/436
* Documentation Rehaul by @adhilto in https://github.com/cisagov/ScubaGoggles/pull/319
* Add instructions for upgrading scubagoggles by @adhilto in https://github.com/cisagov/ScubaGoggles/pull/433
* Update OPA.md by @LaurenBassett in https://github.com/cisagov/ScubaGoggles/pull/446
* Github guideline for branching and release process by @snarve in https://github.com/cisagov/ScubaGoggles/pull/365

## Bugs Fixed 
* Fixing Various Reported Broken Links by @mdueltgen in https://github.com/cisagov/ScubaGoggles/pull/437
* Correct the "No Such Event" behavior for Gmail policies with multiple settings by @adhilto in https://github.com/cisagov/ScubaGoggles/pull/369
* Correct no log event found for Gmail.14.1v03 by @LaurenBassett in https://github.com/cisagov/ScubaGoggles/pull/468

## Baselines
* New Policy 5.1 for Meet for 1:1 Calling by @mdueltgen in https://github.com/cisagov/ScubaGoggles/pull/316
* Changes to Gmail Policies 9.1, 9.2 and 13.1 by @jkaufman-mitre in https://github.com/cisagov/ScubaGoggles/pull/310
* Add policy group 5 in Classroom baseline stating that only verified teachers can create classes by @jkaufman-mitre in https://github.com/cisagov/ScubaGoggles/pull/325
* Remove GWS.CHAT.5.1 by @jkaufman-mitre in https://github.com/cisagov/ScubaGoggles/pull/322
* Added Gmail Policy Group 19 by @jkaufman-mitre in https://github.com/cisagov/ScubaGoggles/pull/324
* Drive_Docs Changes to 1.6 and 6.1 by @jkaufman-mitre in https://github.com/cisagov/ScubaGoggles/pull/311
* Changes to Common Controls Baseline by @jkaufman-mitre in https://github.com/cisagov/ScubaGoggles/pull/315
* Update Gmail policies 5 and 7 to include all instances of spoofing and authentication settings that are not complaint by @snarve in https://github.com/cisagov/ScubaGoggles/pull/394
* Switched Security to Secure Across All Baselines by @jkaufman-mitre in https://github.com/cisagov/ScubaGoggles/pull/389

## Dependency Updates
* The minimum supported Python version is now v3.10.x
* The supported version for the following Python modules has changed:
    * requests: changed from v2.32.0 to v2.32.3
    * tqdm: changed from v4.66.3 to v4.66.5
    * google-api-python-client from v1.7.9 to v2.142.0
    * google-auth-httplib2 from v0.0.3 to v0.2.0
    * google-auth-oauthlib from v0.4.0 to v1.2.1
    * MarkupSafe from v2.1.1 to v2.1.5

## New Contributors
* @aormu made their first contribution in https://github.com/cisagov/ScubaGoggles/pull/356
* @k-winters made their first contribution in https://github.com/cisagov/ScubaGoggles/pull/401

**Full Changelog**: https://github.com/cisagov/ScubaGoggles/compare/v0.2.0...v0.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/cisagov/ScubaGoggles/releases/tag/v0.3.0'>v0.3.0</a>.</em>

**Comment by rob-75** [comment]
I'm new to Python and GitHub. Please can someone tell me where we find the .whl file?

**Reply by rlxdev** [reply]
Hi,

We haven't made a formal release yet for the latest changes reflected in the installation documentation, so there is no `.whl` file available at the moment.  The current version is `0.3.0` and the release is done via a ZIP file.   A formal release (most likely 0.4.0) is coming relatively soon, and I'd recommend that you wait for this release.  A `.whl` will be available for the next release, and the installation instructions will correspond to that release.

**Reply by rob-75** [reply]
Understood. Thanks for the speedy response.
